### ISSUE TITLE
How to save the snapshot of the avd

### ISSUE THREAD

**Issue by cjfcsjt** [google-research/android_world#165]
Running ```env_launcher.load_and_setup_env()``` with ```emulator_setup=True``` requires a few minutes. Is it possible to run the setup and then save the snapshot of the avd?

**Comment by crawles** [comment]
Yes, it’s definitely possible. Do you know where your AVD snapshot is located on your MacBook? On macOS, it's typically in `~/.android/avd/`. You should be able to save this and reload it

Working on a more permanent solution.

**Comment by cjfcsjt** [comment]
Thanks for reply! I have an additional question:

When attempting a specific task multiple times (e.g., 10 times) with the following config:

```
suite = suite_utils.create_suite(
      task_registry.get_registry(family=_SUITE_FAMILY.value),
      n_task_combinations=10, # n_task_combinations,
      seed=2025, # _TASK_RANDOM_SEED.value,
      tasks=['AudioRecorderRecordAudioWithFileName'],
      use_identical_params=True, # _FIXED_TASK_SEED.value,
)
```
I noticed that the environment changes unexpectedly, for example, Record-1 becomes 1Record-1, with an extra 1 added.
![bug_in_aw](https://github.com/user-attachments/assets/3b14828d-2345-484b-8c28-58bfd2e73a14)


How can I resolve this issue?

**Comment by crawles** [comment]
You can set this flag equal to true
https://github.com/google-research/android_world/blob/653a02986f62d8676061510261eb1629fde95dd6/run.py#L133

The reason the task changes is because the random seed changes

Chris

**Comment by crawles** [comment]
Ah, sorry I didn't look closely enough at your code :)

I've figured out the bug and it will be fixed by https://github.com/google-research/android_world/pull/185. Thanks for reporting!
